src/github.com/getlantern/checkfallbacks/checkfallbacks.go

```diff
@@ -149,6 +150,7 @@ func testFallbackServer(fb *client.ChainedServerInfo, workerId int) (output full
 		output.info = []string{"\n" + string(reqStr)}
 	}
 
+	req.Header.Set("X-LANTERN-AUTH-TOKEN", fb.AuthToken)
```

Comment:
balancer never touches anything with HTTP -- it just deals with net.Dial and net.Conn @fffw and @uaalto. That was actually part of the difficulty with adding the auth header more generally outside of using CONNECT for everything in chained/dialer.go the way we used to.